JavascriptFormatter.js

```diff
@@ -44,7 +44,7 @@ function filterForRemoteControl(originalCommands) {
             var c1 = c.createCopy();
             c1.command = c.command.replace(/AndWait$/, '');
             commands.push(c1);
-            commands.push(new Command("waitForPageToLoad", options['global.timeout'] || "30000"));
+            commands.push(new Command("waitForPageToLoad", "options.timeout || " + (options['global.timeout'] || "30000")));
```

Comment:
Still don't know what you mean. I didn't put in options['global.timeout']; it was already there? I was just assuming you needed it for something even though I couldn't see where it came from, so I left it in? :)

Cheers